Project/DAQ_System/Program/Src/Platform/I2C/STM/i2c_stmf4.hpp

```diff
@@ -13,18 +13,18 @@
 
 namespace platform{
 
-class i2cHalWrapperStmf4 : public II2C{
+class i2c_stmf4 : public II2C{
 public:
-	i2cHalWrapperStmf4(I2C_HandleTypeDef &hi2c);
+	i2c_stmf4(I2C_HandleTypeDef &hi2c);
 
-	virtual ~i2cHalWrapperStmf4();
+	virtual ~i2c_stmf4();
 
 	virtual void Receive(uint8_t*, uint8_t, uint8_t) override;
 
-	virtual void Transmit(uint8_t*, uint8_t) override;
+	virtual void Transmit(uint8_t*, uint8_t, uint8_t) override;
```

Comment:
Include the names of the function parameters. The `.hpp` file is usually where a developer starts when examining what a class component does and what are its requirements. We want to be explicit and transparent for the code to work as self-documentation.

**Example**: Looking at `Transmit()`, I would not know the expected order of providing the buffer array, number of bytes to send, etc.

Add parameter names on the `II2c` interface, too.